### PR TITLE
style, more color bins, updated legend

### PR DESCRIPTION
**Pull request by jread-usgs**
image

### CODE REVIEW

File: scripts/visualize/visualize.hurricane_map.R [hunk 1]
```diff
@@ -177,9 +177,10 @@ add_legend <- function(parent.ele, colors, break.step){
   xml_add_child(parent.ele, 'text', "Total rainfall amount (inches)", class='svg-text legend-text', dy="-1em",
                 transform="translate(0,35)")
   g.rains <- xml_add_child(parent.ele, 'g', id = 'rain-legend')
-  g.irma <- xml_add_child(parent.ele, 'g', id = 'irma-legend', transform="translate(15,-65)")
-  g.gage_isFlood <- xml_add_child(parent.ele, 'g', id = 'gage-legend', transform="translate(15,-10)")
-  g.gage_predFlood <- xml_add_child(parent.ele, 'g', id = 'gage-legend', transform="translate(15,-25)")
+  g.irma <- xml_add_child(parent.ele, 'g', id = 'irma-legend', transform="translate(15,-80)")
```
Comment (dblodgett-usgs) [inline scripts/visualize/visualize.hurricane_map.R]: Irma will be with us always.

Comment (jread-usgs) [reply scripts/visualize/visualize.hurricane_map.R]: This legend code needs some clean up.

File: viz.yaml [hunk 2]
```diff
@@ -298,7 +298,7 @@ process:
     scripts: scripts/process/process.precip_colors.R
     reader: rds
     depends: ["precip-colors"]
-    stepSize: 6
+    stepSize: 3
```
Comment (dblodgett-usgs) [inline viz.yaml]: Nice! Thinking we may want a few more labels on this color scale since it's such a broad range?

Comment (jread-usgs) [reply viz.yaml]: agreed.

**Pull request merged**